compute_sdk/globus_compute_sdk/sdk/executor.py

```diff
@@ -545,9 +545,16 @@ def map(fxexec, fn, *fn_args_kwargs):
 
         Raises
         ------
+        RuntimeError
+          if called after shutdown, otherwise, ...
+
         NotImplementedError
-          always raised
+          ... always raised
```

Comment:
I'm not sure what you were trying to say with the '...'.  you do mean  
`RuntimeError 
  if called after shutdown, otherwise, NotImplementedError`
  right?  Unless the multiline format is the standard readability style.

Reply:
Trying to mix them, because they show up as individual items in the documentation.  See screenshot:

<img src="https://github.com/funcx-faas/funcX/assets/99676336/f08c42aa-4d0a-409e-8247-87b1c213390e" width=450>